packages/react/src/components/Modal/Modal.stories.tsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from 'react';
+import { Meta } from '@storybook/react';
+import Modal from './Modal';
+import Button from '../Button';
+
+const meta: Meta<typeof Modal> = {
+  title: 'Components/Modal',
+  component: Modal,
+};
+
+export default meta;
+
+const Template = props => {
+  const [open, setOpen] = useState(false);
+
+  return (
+    <>
+      {
+        open &&
+          <span style={{ position: 'fixed', left: 0, right: 0, top: 0, bottom: 0, background: 'black', opacity: 0.7 }} />
```

Comment:
I think the modal should export this as a component `/components/Modal/Backdrop.tsx` or `Overlay.tsx` or something you like